### DISCUSSION TITLE
next/head not working SSR

### DISCUSSION THREAD

**Discussion by getorca** [vercel/next.js#14373] [Help]
in the next/head example it's rendered server side, visible when viewing source, and inspecting element. However in my it appears to not be, not visible in the source, but visible inspecting element. 

Has anyone else run into this? What was the cause?

**Comment by revskill10** [comment]
Do you have any codesandbox/code snippet to reproduce problem ?

**Reply by getorca** [reply]
I'm trying, that's what I'm figuring a conflict, or experimental feature, so going through now manually to try and recreate.

Hoping someone has experienced the same thing, and maybe knows of a common conflict.

**Comment by umutahmet** [comment]
Was there a conclusion to this? I'm experience the exact same issue. Not sure what's happening.

**Reply by nisargrthakkar** [reply]
I am facing the same issue, do we have any update on same ?

**Reply by maksimf** [reply]
I had similiar issue and [this](https://github.com/vercel/next.js/issues/18021#issuecomment-827407105) what's solved it for me

**Reply by VasilKostadinov0104** [reply]
> I had similiar issue and [this](https://github.com/vercel/next.js/issues/18021#issuecomment-827407105) what's solved it for me

I had that by me and i am confirming - it solved the issue. Thank you!
